quintanalibre.org.ar/generic/files/etc/uci-defaults/97-dropbear

```diff
@@ -1,4 +0,0 @@
-#!/bin/sh
-# Fix dropbear authorized keys permissions
-chmod 0600 /etc/dropbear/authorized_keys
-chmod 0700 /etc/dropbear
```

Comment:
We have lost the permission setup for the /etc/dropbear directory, (not for authorized_keys). Which AFAIU is not a big issue. But I raise my hand to have a second opinion